dspace/modules/additions/pom.xml

```diff
@@ -194,6 +203,35 @@
             </exclusion>
          </exclusions>
       </dependency>
+        <dependency>
+            <groupId>org.apache.lucene</groupId>
+            <artifactId>lucene-core</artifactId>
+            <version>${solr.client.version}</version>
```

Comment:
Same as above, should this have  `<scope>test</scope>` as well?

Reply:
I recall that, once I fixed the above, I had to add it here too, to get `additions` to build.